src/QueryRenderer.ts

```diff
@@ -118,11 +119,37 @@ class QueryRenderChild extends MarkdownRenderChild {
     }
 
     private async render({ tasks, state }: { tasks: Task[]; state: State }) {
+        tasks.map(async (task) => {
+            if (task.description.startsWith('![[')) {
+                const link = parseLinktext(task.description);
+                const subpath = link.subpath
+                    .replace('#^', '')
+                    .replace(']]', '');
+                if (link) {
+                    const file = this.app.metadataCache.getFirstLinkpathDest(
+                        link.path.replace('![[', ''),
+                        task.path,
+                    );
+                    const content =
+                        file && (await this.app.vault.read(file)).split('\n');
+                    const blocks =
+                        file &&
+                        this.app.metadataCache.getFileCache(file)?.blocks;
+                    const line = blocks && blocks[subpath]?.position.start.line;
+                    if (line) {
+                        task.description = content[line]
+                            .split(' ^')[0]
+                            .replace('- ', '');
+                    }
+                }
+            }
+        });
```

Comment:
@shabegom can you please explain how this is related to the custom date format?

Reply:
This is so weird! This is from a totally unrelated plugin (block ref counter) and I have no idea how it made it in here. My best guess is Github Copilot suggested this code and I hit tab by accident...

I'm sorry I didn't catch this before. That's really embarrassing. can be fully deleted.

Reply:
No worries